tests/integration/components/rental-listing-test.js

```diff
@@ -0,0 +1,18 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('rental-listing', 'Integration | Component | rental listing', {
+  integration: true
+});
+
+test('it renders', function(assert) {
+  // assert.expect(2);
+
+  // Set any properties with this.set('myProperty', 'value');
+  // Handle any actions with this.on('myAction', function(val) { ... });
+
+  this.render(hbs`{{rental-listing}}`);
```

Comment:
Since its the default generated test from ember-cli, can we delete the file?  I can do a separate PR with some more valid tests.  I think that would be a good example.